### PR TITLE
Deleted poorly formatted docstrings in the CLI and improved the help …

### LINKED ISSUE #745: [Bug] Help text for CLI commands is not properly formatted
**Describe the bug**
The help text for CLI commands is weirdly formatted.

Example: `vnode attach --help` gives:

```
Usage: vnode attach [OPTIONS]

  Attach the logs from the docker container to the terminal.

  Parameters ---------- name : str     Name of the configuration file.
  system_folders : bool     Wether this configuration stored in the system or
  in the user folders.

Options:
  -n, --name TEXT  configuration name
  --system
  --user
  --help           Show this message and exit.
```

- [x] Wait for #624 to start this to prevent merge conflicts
- [x] Remove the Parameters section, instead add help to the relevant options
- [x] Check the docs to see if this is properly displayed (does not seem to be the case in live version!)

### PR DESCRIPTION
**Pull request by bartvanb**
Fix #745

### CODE REVIEW

File: vantage6/vantage6/cli/dev.py [hunk 1]
```diff
@@ -293,7 +293,10 @@ def demo_network(num_nodes: int, server_url: str, server_port: int,
 
 @click.group(name="dev")
 def cli_dev() -> None:
-    """Subcommand `vdev`."""
+    """
+    The `vdev` commands can be used to quickly manage a development network
```
Comment (frankcorneliusmartin) [inline vantage6/vantage6/cli/dev.py:297]: development/test

File: vantage6/vantage6/cli/node.py [hunk 2]
```diff
@@ -231,26 +224,34 @@ def cli_node_files(name: str, environment: str, system_folders: bool) -> None:
 #   start
 #
 @cli_node.command(name='start')
-@click.option("-n", "--name", default=None, help="configuration name")
+@click.option("-n", "--name", default=None, help="Configuration name")
 @click.option("-c", "--config", default=None,
-              help='absolute path to configuration-file; overrides NAME')
+              help='Path to configuration-file; overrides NAME')
```
Comment (frankcorneliusmartin) [inline vantage6/vantage6/cli/node.py:229]: Did you check that a relative path also works?

**Review by frankcorneliusmartin — APPROVED** [review]

File: vantage6/vantage6/cli/node.py (hunk 2 above)
Comment (bartvanb) [reply vantage6/vantage6/cli/node.py:229]: You're right, it doesn't. Changed it back

File: vantage6/vantage6/cli/dev.py (hunk 1 above)
Comment (bartvanb) [reply vantage6/vantage6/cli/dev.py:297]: I just got rid of the word entirely since it is also in the next line (same sentence) so it was double

**Pull request merged**